### PR TITLE
Vlad Nazanin Ulyssess

### PR DESCRIPTION
**Pull request by nazi-pira**
Link to my deployed project:
https://brave-colden-781fca.netlify.app/
My reflections on how this week's project turned out:
This weeks project was quite challenging and we have learnt a lot from each other making it!

### CODE REVIEW

File: code/src/components/GameScreen.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+                <h3>{path.description}</h3>
+                <h4>Do you dare to go:</h4>
+                <div className="directions">
+                    {path.actions.map((action) => (
+                        // <button
+                        //   type="button"
+                        //   className="nes-btn is-primary"
+                        //   key={action.direction}
+                        //   onClick={() => onPathChoice(action.direction)}>{action.direction}
+                        // </button>
```
Comment (Bellsibub) [inline code/src/components/GameScreen.js:35]: ❗ commented out code should be removed 😉

File: code/src/components/gameScreen.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  color: white;
+  border: 3px solid white;
+  padding: 5px 10px;
+  position: relative;
+  transition: all 1s;
+  overflow: hidden;
+}
+
+.btn-back:before {
+  content: '🐔';
```
Comment (Bellsibub) [inline code/src/components/gameScreen.css:58]: LOL 🤣

File: code/src/components/paths.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  }
+}
+@keyframes spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```
Comment (Bellsibub) [inline code/src/components/paths.css:36]: Very nice 👍

File: code/src/components/startScreen.css [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+.start-btn:hover span:after {
+  bottom: 0;
+}
+
+.start-btn:hover {
+  background-color: rgb(5, 7, 107);
+  color: rgb(245, 7, 67);
+  font-weight: 800;
+  transition: all 3s;
+}
```
Comment (Bellsibub) [inline code/src/components/startScreen.css:129]: Amazing 😍

File: code/src/components/startScreen.css [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+  .start-game {
+      background-size: cover;
+      position: absolute;
+      align-items: center;
+      justify-content: center;
+  }
+  
+  .start-wrapper {
+     
+  }
```
Comment (Bellsibub) [inline code/src/components/startScreen.css:159]: ❗ The empty `.start-wrapper` class should be removed

**Review by Bellsibub — CHANGES_REQUESTED** [review]
Great Project! 🥇 
Very clean and easy to follow and read.

However there were two issues I found in the code (super minor!) but nevertheless required to be fixed I think 😉 
I marked those requested changes with an ❗ icon. They can be found in the following files:
- GameScreen.js
- startScreen.css

File: code/src/components/startScreen.css (hunk 4 above)
Comment (KarolCislowski) [reply code/src/components/startScreen.css:129]: Awesome but font color is not very good 😢 
image

**Review by KarolCislowski — APPROVED** [review]
Nice app 🥳

**Pull request closed without merging**